src/services/index.js

```diff
@@ -33,12 +33,20 @@ const rocketChat = require('./rocketChat');
 const clipboard = require('./clipboard');
 const me = require('./me');
 
-module.exports = function () {
+const newsEvents = require('./news/events');
+
+module.exports = function initializeServices() {
 	const app = this;
 
-	mongoose.connect(process.env.DB_URL || app.get('mongodb'), { user: process.env.DB_USERNAME, pass: process.env.DB_PASSWORD });
+	// connect mongoose to the database
+	const dbCredentials = {
+		user: process.env.DB_USERNAME,
+		pass: process.env.DB_PASSWORD,
+	};
+	mongoose.connect(process.env.DB_URL || app.get('mongodb'), dbCredentials);
```

Comment:
this string appears in a lot of times and should be moved to a single configuration file

Reply:
true. This file is only part of the PR due to linting ;-)